types/geojson/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: http://geojson.org/
 // Definitions by: Jacob Bruun <https://github.com/cobster>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
why do you need this?

Reply:
It caused a compile issue:

```
$ npm run lint geojson

> definitely-typed@0.0.1 lint /Users/mac/Github/DefinitelyTyped
> dtslint types "geojson"

Test with 2.4
Test with 2.3
Test with 2.2
Test with 2.1
/Users/mac/Github/DefinitelyTyped/types/geojson/geojson-tests.ts
ERROR: 222:15  expect  Compile error in typescript@2.1 but not in typescript@2.2.
Fix with a comment '// TypeScript Version: 2.2' just under the header.
Property 'foo' does not exist on type '{ [key: string]: any; }'.
ERROR: 223:15  expect  Compile error in typescript@2.1 but not in typescript@2.2.
Fix with a comment '// TypeScript Version: 2.2' just under the header.
Property 'hello' does not exist on type '{ [key: string]: any; }'.
```
I'll switch the `properties` type to `any` instead.